src/engraving/tests/compat114_data/fingering-ref.mscx

```diff
@@ -72,7 +72,7 @@
       </Part>
     <Staff id="1">
       <VBox>
-        <height>10</height>
+        <height>0</height>
```

Comment:
why 0?

Reply:
Ah I think this is a mistake - looking into it